src/PointwiseFunctions/AnalyticData/GrMhd/OrszagTangVortex.hpp

```diff
@@ -0,0 +1,162 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+// IWYU pragma: no_include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"  // for IdealFluid
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace AnalyticData {
+
+/*!
+ * \brief Analytic initial data for the relativistic Orszag-Tang vortex.
+ *
+ * The relativistic version of the Orszag-Tang vortex is a
+ * 2-dimensional test case for relativistic MHD systems (see, e.g.,
+ * [arXiv:1101.3573](https://arxiv.org/abs/1101.3573)
+ * \cite Beckwith2011).  It describes the flow of an ideal fluid with
+ * adiabatic index \f$5/3\f$.  The initial conditions (and hence the
+ * states at later times) are periodic in both \f$x\f$ and \f$y\f$
+ * with period 1.  The initial conditions are:
+ * \f{align*}
+ * \rho &= \frac{25}{36 \pi} \\
+ * p &= \frac{5}{12 \pi} \\
+ * v_x &= -\frac{1}{2} \sin(2 \pi y) \\
```

Comment:
There could easily be a subtlety I'm missing, but at first glance it seems that the velocity x-component has a sign change as compared to the cited paper.

Reply:
I think it would be helpful to have the small note that their paper is also slightly incorrect, to minimize confusion if this is revisited sometime in the future.